MongoDB/containers/0.1.0/mongodb-config/entrypoint.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Installing dnsutils and jq
+echo "deb http://http.debian.net/debian wheezy-backports main" | tee /etc/apt/sources.list.d/wheezy-backports.list > /dev/null
+echo "deb http://security.debian.org/ wheezy/updates main contrib non-free " | tee /etc/apt/sources.list.d/wheezy-security.list > /dev/null
+echo "deb-src http://security.debian.org/ wheezy/updates main contrib non-free" | tee /etc/apt/sources.list.d/wheezy-security.list > /dev/null
+apt-get -q update > /dev/null
+apt-get install -qqy dnsutils jq > /dev/null 2>&1
+
+# Check for lowest ID
+/opt/rancher/bin/lowest_idx.sh
```

Comment:
@galal-hussein, you can get around needing it to run in the MongoDB container, by running mongod-conf with `net: "container:mongo-cluster"` (Thoughts on name? should it just be mongodb since when running it will be stack_mongo-cluster_1 I'm neutral on it ) Running the conf container in the servers net namespace makes DNS and Metadata  have the same view in both containers. 

The entrypoint will look more like this:

```
while [ ! -f /iplist ]; then
 sleep 5
fi
```

while the conf container does the digging.

jq is small, so I think its ok. I think lowest_idx.sh will work in the shared net namespace, there will be 3 fewer entries. Probably wait on the result in the entrypoint, otherwise you could have a race condition.